randovania/games/dread/patcher/open_dread_patcher.py

```diff
@@ -197,3 +259,7 @@ def patch_game(self, input_file: Optional[Path], output_file: Path, patch_data:
         with output_file.joinpath("patcher.json").open("w") as f:
             json.dump(patch_data, f, indent=4)
         open_dread_rando.patch(input_file, output_file, patch_data)
+
+class DreadAcquiredMemo(dict):
```

Comment:
Was this so we could change the text later on?

Reply:
it differs from the prime message in exactly one way: dread acquisitions end with a period, not an exclamation mark

Reply:
Oh. Though I'm planning on changing these to match the vanilla style of the popups anyway

Reply:
this implementation matches the vanilla style in dread

Reply:
Not for expansions :)

Reply:
ah, fair. can just add those to the dict ofc